Gruntfile.js

```diff
@@ -36,7 +36,7 @@ module.exports = function(grunt) {
       }
     },
     concurrent: {
-      serve: ['nodemon', 'watch'],
+      serve: ['nodemon'],
```

Comment:
I don't think we should remove the watch, other templates have it and can be useful.